cdap-ui/app/cdap/components/PipelineDetails/RunLevelInfo/RunConfigs.js

```diff
@@ -98,7 +100,10 @@ export default class RunConfigs extends Component {
 
   isRuntimeArgsEmpty = () => {
     if (this.state.runtimeArgs.pairs.length === 1) {
-      if (isNilOrEmpty(this.state.runtimeArgs.pairs.key) && isNilOrEmpty(this.state.runtimeArgs.pairs.value)) {
+      if (
+        isNilOrEmpty(this.state.runtimeArgs.pairs[0].key) &&
+        isNilOrEmpty(this.state.runtimeArgs.pairs[0].value)
```

Comment:
should this condition be an `OR` instead? I think our count somewhere would omit those with empty key or value

Reply:
Since this is for historical runs this check is to only check if the value in is the default value. 
```
{
  pairs: [{
    key: '',
    value: ''
  }]
}
```
This the value given by the keyvalue pairs component(or the store). The correctness of this default value seems incorrect but this is to check for that. We get this value from the run record.